docs/development-setup.md

```diff
@@ -1,73 +1,158 @@
 # Local Development
 
-It's actually very simple to setup OpenShift locally. We'll use a docker image from Docker Hub. For more OpenShift-focused information, see [this guide](https://docs.openshift.org/latest/getting_started/administrators.html).
 
-First thing to do is to clone this git repo:
+## Running OpenShift
+
+For more options how to install and run OpenShift, please see [this page](https://install.openshift.com/).
+
+We'll use the `oc cluster up` method, since it's very easy to use:
+
+ 1. If you're on Fedora, you can install package origin:
+    ```
+    dnf install -y origin
+    ```
+    Now you have command `oc` available on your system
+
+ 2. You can start a single-node cluster like this
+    ```
+    oc cluster up
+    ```
+    (If you are running into some connection issues, please flush your iptables
+    rules bufore starting the cluster with `iptables -F`)
+
+
+## Getting `osbs-client`
+
+For local development, we advise to clone this git repo:
 
 ```
 $ git clone git@github.com:projectatomic/osbs-client.git
 $ cd osbs-client
```

Comment:
Maybe add `python setup.py develop` for functional parity with `dnf` approach below?